[2025-09-06T07:43:44.355+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-06T07:43:44.380+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_customers_to_postgres.show_filepath manual__2025-09-06T07:43:43.092994+00:00 [queued]>
[2025-09-06T07:43:44.385+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_customers_to_postgres.show_filepath manual__2025-09-06T07:43:43.092994+00:00 [queued]>
[2025-09-06T07:43:44.385+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-06T07:43:44.392+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): show_filepath> on 2025-09-06 07:43:43.092994+00:00
[2025-09-06T07:43:44.396+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'load_customers_to_postgres', 'show_filepath', 'manual__2025-09-06T07:43:43.092994+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/load_customers_to_postgres.py', '--cfg-path', '/tmp/tmp961p7_uk']
[2025-09-06T07:43:44.399+0000] {standard_task_runner.py:91} INFO - Job 2: Subtask show_filepath
[2025-09-06T07:43:44.398+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=184) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-06T07:43:44.400+0000] {standard_task_runner.py:64} INFO - Started process 185 to run task
[2025-09-06T07:43:44.434+0000] {task_command.py:426} INFO - Running <TaskInstance: load_customers_to_postgres.show_filepath manual__2025-09-06T07:43:43.092994+00:00 [running]> on host d3743a3a0ea9
[2025-09-06T07:43:44.491+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='etl' AIRFLOW_CTX_DAG_ID='load_customers_to_postgres' AIRFLOW_CTX_TASK_ID='show_filepath' AIRFLOW_CTX_EXECUTION_DATE='2025-09-06T07:43:43.092994+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-06T07:43:43.092994+00:00'
[2025-09-06T07:43:44.492+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-06T07:43:44.493+0000] {logging_mixin.py:188} INFO - ðŸ‘€ Looking for file at: /opt/***/data/customer_churn_data.csv
[2025-09-06T07:43:44.493+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/data/customer_churn_data.csv
[2025-09-06T07:43:44.493+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-06T07:43:44.505+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=load_customers_to_postgres, task_id=show_filepath, run_id=manual__2025-09-06T07:43:43.092994+00:00, execution_date=20250906T074343, start_date=20250906T074344, end_date=20250906T074344
[2025-09-06T07:43:44.545+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-06T07:43:44.558+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-06T07:43:44.559+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
